client/src/components/UsedOnTable/UsedOnTable.js

```diff
@@ -64,23 +63,24 @@ class UsedOnTable extends PureComponent {
       id,
       title,
       type,
-      state,
       link,
     } = data;
 
-    const badge = (state)
-      ? <span className={classnames('badge', 'used-on__badge', `status-${state}`)}>{state}</span>
-      : null;
-
     const titleLabel = (link)
-      ? <a className="used-on__edit-link" href={link}>{title} {badge}</a>
-      : <span>{title} {badge}</span>;
+      ? <a className="used-on__edit-link" href={link}>{title}</a>
```

Comment:
This approach is fine, but we could have made this a simple functional component which would have produce cleaner code. Something to keep in mind next time.